tardis/io/schemas/csvy_model.yml

```diff
@@ -0,0 +1,147 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: object
+additionalProperties: false
+properties:
+  name: 
+    type: string
+    description: name of the model being run
+
+  description:
+    type: string
+    description: description of the model being run
+
+  tardis_model_config_version:
+    type: string
+    description: Version of the configuration file
+
+  atom_data:
+    type: string
+    description: path to the atom data
+
+  datatype:
+    type: object
+    additionalProperties: False
+    properties:
+      fields:
+        type: array
+
+  v_inner_boundary:
+    type: quantity
+    description: velocity of the inner boundary
+
+  v_outer_boundary:
+    type: quantity
+    description: velocity of the outer boundary of the last shell
+
+  velocity:
+    type: object
+    properties:
+      start:
+        type: quantity
+      stop:
+        type: quantity
+      num:
+        type: number
+        multipleOf: 1.0
+
+  density:
+    oneOf:
+    - $ref: '#/definitions/density/branch85_w7'
+    - $ref: '#/definitions/density/exponential'
+    - $ref: '#/definitions/density/power_law'
+    - $ref: '#/definitions/density/uniform'
+
+  abundance:
+    $ref: '#/definitions/abundance/uniform'
+
+  required:
+  - name
```

Comment:
Do we want to require a name for the model?

Reply:
nope - we should not. can the required key live within the node

Reply:
I'm pretty sure that the required key cannot live within the node.

https://json-schema.org/understanding-json-schema/reference/object.html#required